tests/Kernel/fixtures/rendering.yml

```diff
@@ -2342,14 +2365,17 @@
     count:
       'div.ecl-row': 0
       'figure.ecl-media-container': 0
+      'div.ecl-col-md-6.ecl-u-mb-m.ecl-u-mb-md-none': 0
```

Comment:
When no variant is passed, we should make sure the left_simple is used so we could just add the link.

Reply:
This particular test ensures that content is empty. 
I've added case with empty variant below.